spatialdata/_core/_transform_elements.py

```diff
@@ -197,7 +198,7 @@ def _(data: DaskDataFrame, transformation: BaseTransformation) -> DaskDataFrame:
     axes = get_dims(data)
     arrays = []
     for ax in axes:
-        arrays.append(data[ax].to_dask_array())
+        arrays.append(data[ax].to_dask_array(lengths=True))
```

Comment:
Without this, I was having issues with the chunk size being unknown for subsequent operations on the transformed points.

Reply:
thanks for the info